hoomd_validation/patchy_particle_pressure.py

```diff
@@ -0,0 +1,820 @@
+# Copyright (c) 2022-2023 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+"""Test for consistency between NVT and NPT simulations of patchy particles."""
+
+from config import CONFIG
+from project_class import Project
+from flow import aggregator
+import util
+import os
+import json
+import pathlib
+
+# Run parameters shared between simulations.
+# Step counts must be even and a multiple of the log quantity period.
+RANDOMIZE_STEPS = 20_000
+EQUILIBRATE_STEPS = 200_000
+RUN_STEPS = 500_000
+RESTART_STEPS = RUN_STEPS // 50
+TOTAL_STEPS = RANDOMIZE_STEPS + EQUILIBRATE_STEPS + RUN_STEPS
+
+WRITE_PERIOD = 1_000
+LOG_PERIOD = {'trajectory': 50_000, 'quantities': 500}
+NUM_CPU_RANKS = min(16, CONFIG["max_cores_sim"])
+
+WALLTIME_STOP_SECONDS = CONFIG["max_walltime"] * 3600 - 10 * 60
+
+
+def job_statepoints():
+    """list(dict): A list of statepoints for this subproject."""
+    num_particles = 16**3
+    replicate_indices = range(CONFIG["replicates"])
+    params_list = [
+        # kern-frenkel statepoints. densities/temperatures chosen to prorduce
+        # a dense liquid. pressures measure from NVT simulations.
+        # (10.0, 0.95, 11.951630531873338, 0.7, 1.5, 1.0),
+        (1.0, 0.95, 10.208625410213362, 0.7, 1.5, 1.0),
+        # (0.6, 0.95, 8.927827449359, 0.7, 1.5, 1.0),
+        # hard sphere + square well statepoints, from 10.1063/1.3054361
+        # pressure from NVT simulations, NOT from values in paper
+        # (0.5714, 0.8, -0.2692376274894095, 1.0, 1.5, 1.0),
+        # (1.0, 0.8, 2.2766339608381325, 1.0, 1.5, 1.0),
+        (3.0, 0.8, 4.837436833423719, 1.0, 1.5, 1.0),
+        # hard sphere + square well + repulsive shoulder statepoints.
+        # temperatures/densities from initial tests, pressures from NVT
+        # simulations.
+        (3.0, 0.7, 4.00804, 1.0, 1.0, -1.0),
+        # (1.0, 0.7, -5.0, 1.0, 1.0, -1.0),
```

Comment:
```suggestion
        (1.0, 0.95, 10.208625410213362, 0.7, 1.5, 1.0),
        # hard sphere + square well statepoints, from 10.1063/1.3054361
        # pressure from NVT simulations, NOT from values in paper
        (3.0, 0.8, 4.837436833423719, 1.0, 1.5, 1.0),
        # hard sphere + square well + repulsive shoulder statepoints.
        # temperatures/densities from initial tests, pressures from NVT
        # simulations.
        (3.0, 0.7, 4.00804, 1.0, 1.0, -1.0),
```
Remove unused state points.